src/listeners/decorators/once.decorator.ts

```diff
@@ -0,0 +1,13 @@
+import { Listener } from 'necord';
+import { LavalinkManagerEvents } from 'lavalink-client/dist/types/structures/Types/Manager';
+import { NodeManagerEvents } from 'lavalink-client';
+
+/**
+ * Decorator that marks a method as a listener for Lavalink client.
+ * @param event The event name.
+ * @returns The decorated method.
+ * @url https://necord.org/recipes/lavalink/listeners
+ */
+export const OnceLavalink = <K extends keyof E, E = LavalinkManagerEvents | NodeManagerEvents>(
```

Comment:
Also not sure about global enhancers they can affect on `Listener`, cuz it connected via ExternalContextCreator

Reply:
so it would be better to create a new decorator to not use Necord's Listerner?

Reply:
LGTM

Reply:
> so it would be better to create a new decorator to not use Necord's Listerner decorator?

Yeah, I think it will be better also I think it not need support by enhancers for some reason

Reply:
What you mean by "support by enhancers"

Reply:
Enhancers - Guards, Interceptors, Filters and etc. I think you dont need to do this for lavalink

Reply:
Listener Decorator: https://github.com/necordjs/lavalink/pull/5/commits/199670bccdbb51751134476fc484b560a10af5f0

> Enhancers - Guards, Interceptors, Filters and etc. I think you dont need to do this for lavalink
 
I think the same in this sense